vnext/Microsoft.ReactNative/Views/FocusZoneViewManager.cpp

```diff
@@ -0,0 +1,151 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "pch.h"
+
+#include "FocusZoneViewManager.h"
+
+#include "ViewControl.h"
+
+#include "DynamicAutomationProperties.h"
+
+#include <Modules/NativeUIManager.h>
+#include <Utils/AccessibilityUtils.h>
+#include <Utils/Helpers.h>
+#include <Utils/PropertyUtils.h>
+
+#include <INativeUIManager.h>
+#include <IReactInstance.h>
+
+#include <winrt/Windows.System.h>
+#include <winrt/Windows.UI.Core.h>
+
+#include <UI.Xaml.Controls.Primitives.h>
+#include <UI.Xaml.Input.h>
+#include <winrt/Windows.ApplicationModel.Resources.Core.h>
+
+namespace winrt {
+using namespace Windows::Foundation;
+using namespace Windows::UI;
+using namespace Windows::UI::Xaml;
+using namespace Windows::UI::Xaml::Controls;
+using namespace Windows::UI::Xaml::Controls::Primitives;
+using namespace Windows::UI::Xaml::Media;
```

Comment:
don't use Windows::UI::Xaml directly, only reference the aliased location (xaml, xaml::Controls, etc.), otherwise this will break WinUI 3 builds of RNW.

Reply:
also no need to re-alias these into winrt namespace

Reply:
Sorry, I missed these when updating the namespaces around method calls below.